src/main/java/com/vinylteam/vinyl/entity/Vinyl.java

```diff
@@ -0,0 +1,78 @@
+package com.vinylteam.vinyl.entity;
+
+import java.util.Objects;
+
+public class Vinyl {
+
+    private String release;
+    private String artist;
+    private String price;
+    private String vinylLink;
+    private String imageLink;
+    private String genre;
+
+    public void setRelease(String release) {
+        this.release = release;
+    }
+
+    public void setArtist(String artist) {
+        this.artist = artist;
+    }
+
+    public void setPrice(String price) {
+        this.price = price;
+    }
+
+    public void setVinylLink(String vinylLink) {
+        this.vinylLink = vinylLink;
+    }
+
+    public void setImageLink(String imageLink) {
+        this.imageLink = imageLink;
+    }
+
+    public void setGenre(String genre) {
+        this.genre = genre;
+    }
+
+    public String getRelease() {
+        return release;
+    }
+
+    public String getArtist() {
+        return artist;
+    }
+
+    public String getPrice() {
+        return price;
+    }
+
+    public String getVinylLink() {
+        return vinylLink;
+    }
+
+    public String getImageLink() {
+        return imageLink;
+    }
+
+    public String getGenre() {
+        return genre;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
I think it'd be better for both of these to write
if (...) {
return ...;
}

For better readability? @AlexKievDeveloper , what do you think?

Reply:
I think that you are absolutely right. We should always use bracers {} after if because it will safe us from making mistake in future while code modifying.